src/vm/exceptionhandling.cpp

```diff
@@ -4424,7 +4424,7 @@ VOID UnwindManagedExceptionPass2(EXCEPTION_RECORD* exceptionRecord, CONTEXT* unw
                 dispatcherContext.FunctionEntry,
                 callerFrameContext,
                 &handlerData,
-                &establisherFrame,
+                (SIZE_T*)&establisherFrame,
```

Comment:
It seems we could actually declare the establisherFrame as SIZE_T, or maybe better as UINT_PTR (since that's the type of the targetFrameSp parameter of the UnwindManagedExceptionPass2) and remove this cast. 
If you do this, then please also modify the declaration of the establisherFrame in the UnwindManagedExceptionPass2.